Terracon Quest listing on dappland.com

Agathonas opened the pull request:
Uploaded images, gallery and nft previews of the game.

graemeblackwood commented:
@Agathonas Thanks for this! Please update the painting-style images to jpgs and run through tinyjpg.com. They'll be much smaller but just as good quality.

Also the references are broken (pointing to terracon-quest vs terraconquest.

You can check the Vercel build preview to see what I mean

Agathonas commented:
Thank you for your guidance.
 
I changed all to jpg, compressed them and renamed in correct format. 
 
Thank you.
Terracon Quest Team

> On 07/12/2024 3:17 PM EEST Graeme Blackwood ***@***.***> wrote:
>  
>  
> 
>  
> 
> @Agathonas https://github.com/Agathonas Thanks for this! Please update the painting-style images to jpgs and run through tinyjpg.com. They'll be much smaller but just as good quality.
> 
> Also the references are broken (pointing to terracon-quest vs terraconquest.
> 
> You can check the Vercel build preview to see what I mean
> 
> —
> Reply to this email directly, view it on GitHub https://github.com/argentlabs/dappland/pull/279#issuecomment-2225459650, or unsubscribe https://github.com/notifications/unsubscribe-auth/BFZ3E7JRPEEV7QK3CA6DXVDZL7CNRAVCNFSM6AAAAABKYRPSH2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEMRVGQ2TSNRVGA.
> You are receiving this because you were mentioned.Message ID: ***@***.***>
>
